test-resources/swift/cleanup_rules/input/SampleClass.swift

```diff
@@ -214,6 +214,62 @@ class SampleClass {
         var value2 =  placeholder_false ? 2 : 3
     }
 
+    func checkIfBooleanWithComments(){
+         if !TestEnum.stale_flag_one.isEnabled {
+            // to be deleted
+            toBeDeleted1()
+         }else{
+            // to be preserved
+            toBePreserved1()
+         }
+
+         if !TestEnum.stale_flag_one.isEnabled{
+            // to be deleted
+            toBeDeleted2() 
+         } else if a {
+            // to be preserved
```

Comment:
```suggestion
            // to be preserved 2
```

similarly for others for the sake of consistency